docs/cheatsheet/cheat_sheet_gen.clj

```diff
@@ -98,7 +104,7 @@
 
 (def cheat-sheet-applet
   "
-\\colouredbox{blue1}{
+\\colouredbox{blue}{
   \\section{Applet}
   \\begin{tabularx}{\\hsize}{lX}
  Creation & \\cmd{applet defapplet} \\\\
```

Comment:
Can you update Applet section? I'd suggest to name it "Sketch options" instead. Also functions from "Creation" and "Control" are no longer available so they can be removed completely. Also add `:mouse-wheel` to "Mouse Callback Keywords".